files/zh-cn/web/javascript/reference/deprecated_and_obsolete_features/index.md

```diff
@@ -107,8 +107,8 @@ slug: Web/JavaScript/Reference/Deprecated_and_obsolete_features
 
 | 属性                                                                                     | 描述                                                             |
 | ---------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
-| {{jsxref("Global_Objects/Object/count", "__count__")}}                 | 返回用户自定义对象上的可枚举自身属性的个数。                     |
-| {{jsxref("Global_Objects/Object/Parent", "__parent__")}}             | 指向一个对象的上下文。                                           |
+| {{jsxref("Global_Objects/Object/count", "**count**")}}                 | 返回用户自定义对象上的可枚举自身属性的个数。                     |
+| {{jsxref("Global_Objects/Object/Parent", "**parent**")}}             | 指向一个对象的上下文。                                           |
```

Comment:
```suggestion
| `__count__`                 | 返回用户自定义对象上的可枚举自身属性的个数。                     |
| `__parent__`             | 指向一个对象的上下文。                                           |
```

![image](https://user-images.githubusercontent.com/15844309/204122730-98a9aa04-d92b-487d-bf44-ca0947f3d471.png)

Reply:
Sorry, I missed that one on this PR. On the others I sync'd with the English text. Are you OK if I wipe out the translations, since the text has changed? Otherwise fell free to land your change

Reply:
That's ok, thanks for your help